app/models/foreman_openscap/policy.rb

```diff
@@ -108,10 +110,7 @@ def step_to_i(step_name)
     end
 
     def steps
-      base_steps = [N_('Deployment Options'), N_('Policy Attributes'), N_('SCAP Content'), N_('Schedule')]
-      base_steps << N_('Locations') if SETTINGS[:locations_enabled]
-      base_steps << N_('Organizations') if SETTINGS[:organizations_enabled]
-      base_steps << N_('Hostgroups') # always be last.
+      STEPS_LIST
```

Comment:
Since this is no longer dynamic, no point in recreating the array every time `steps` is called